nxdrive/client/uploader/direct_transfer.py

```diff
@@ -33,10 +33,15 @@ def exists(self, parent_ref: str, name: str) -> bool:
         )
         return bool(self.remote.query(query)["entries"])
 
-    def get_upload(self, file_path: Path) -> Optional[Upload]:
+    def get_upload(
+        self, file_path: Path = None, doc_pair: int = None
+    ) -> Optional[Upload]:
         """Retrieve the eventual transfer associated to the given *file_path*."""
-        ret: Optional[Upload] = self.dao.get_dt_upload(path=file_path)
-        return ret
+        if not file_path and not doc_pair:
+            return None
+        if doc_pair:
+            return self.dao.get_dt_upload(doc_pair=doc_pair)
+        return self.dao.get_dt_upload(path=file_path)
```

Comment:
Am I right if I say we can reduce the entire body to only that?
```suggestion
        return self.dao.get_dt_upload(doc_pair=doc_pair)
```
Because DT transfers always have the `doc_pair` set I think.